ansible/roles/debops.etckeeper/defaults/main.yml

```diff
@@ -140,6 +140,17 @@ etckeeper__default_gitignore:
       vulnerability in case the /etc/ repository is pushed to an external remote.
     ignore: 'keys/mandos/seckey.txt'
 
+  - name: 'borgmatic'
+    comment: |
+      The borgmatic configuration directory can contain sensitive credentials
+      allowing access to backups of the system and potentially other systems as
+      well. debops-contrib.borgbackup only stores credentials in
```

Comment:
If you plan to merge the `borgbackup` role into the monorepo, you can just go with `debops.borgbackup` here. The `debops-contrib.*/` naming was done when the roles were imported from the Contrib organization, but they are not integrated yet with the main playbook. New roles can be just named `debops.*/`.